content/en/apps/guides/hosting/app-developer.md

```diff
@@ -0,0 +1,95 @@
+
+```shell script
+CHT_HTTP=8081 CHT_HTTPS=8443 docker-compose -p the_second -f docker-compose-developer.yml up
+```
+
+The second instance is now accessible at  [https://localhost:8443](https://localhost:8443).
+
+## The `.env` file 
+
+Often times it's convenient to use revision control, like GitHub, to store and publish changes in a CHT app.  A nice compliment to this is store the specifics on how to run the `docker-compose` command for each app. By using a shared `docker-compose` configuration for all developers on the same app, it avoids any port collisions and enables all developers to have a unified configuration.
+
+Using the above `the_second` sample project, we can create another directory to host this project's configuration:
+
+```shell
+mkdir ../the_second
+```
+
+Create a file `../the_second/.env-docker-compose` with this contents:
+
+```shell
+COMPOSE_PROJECT_NAME=the_second
+CHT_HTTP=8081
+CHT_HTTPS=8443
+```
+
+Now it's easy to boot this environment by specifying which `.env` file to use:
+
+```shell
+docker-compose --env-file ../the_second/.env-docker-compose -f docker-compose-developer.yml up
+```
+
+## Switching & concurrent projects
+
+The easiest way to switch between projects is to stop the first set of containers and start the second set. Cancel the first project running in the foreground with `ctrl + c`. Then start the second project using either the `.env` file or use the explicit command with ports and project name as shown above.
+
+To run projects concurrently, instead of cancelling the first one, open a second terminal and start the second project.  
+
+To read more about how `docker-compose` works, be sure to read the [helpful docker-compose commands]({{< relref "core/guides/docker-setup#helpful-docker-commands" >}}) page. 
+
+## Cookie collisions
+
+The CHT stores its cookies based on the domain.  This means if you're running two concurrent instances on `https://localhost` and `https://localhost:8443`, the CHT would store the cookie under the same `localhost` domain. When logging out of one instance, you would get logged out of both and other consistencies.
+
+To avoid this collision of cookies, you can use different IP addresses to access the instances.  This works because the IPs that are available to reference `localhost` are actually a `/8` netmask, meaning [there are 16 million addresses](https://en.wikipedia.org/wiki/Localhost#Name_resolution) to choose from!  
+
+Using the above two configurations, these URLs could work to avoid the cookie colission: 
+
+* [https://127.0.0.1](https://127.0.0.1)
+* [https://127.0.0.2:8443](https://127.0.0.2:8443)
+
+This would result in the domains being `127.0.0.1` and `127.0.0.2` from the CHT's perspective.
+
+Another work around is to use incognito mode, [browser containers](https://support.mozilla.org/en-US/kb/containers) or different browsers.
```

Comment:
Unless it has changed recently. The incognito modes didn't play well with our app due to local storage access.

Reply:
D'oh!  Nice catch - thanks for letting me know.  fixed in 524bfd2